zul/src/archive/web/zul/less/_reset.less

```diff
@@ -6,26 +6,27 @@
 
 // Display in IE6-9 and FF3
 // -------------------------
-
-article,
-aside,
-details,
-figcaption,
-figure,
-footer,
-header,
-hgroup,
-nav,
-section {
+@embeddedCSS: "'org.zkoss.zk.embedded.css'";

```

Comment:
it seems like we have had this property - `@browserDefault: "'org.zkoss.zul.theme.browserDefault'";`
to reset all the overriding CSS for browser since zk 3.6.0 - https://www.zkoss.org/wiki/ZK_Configuration_Reference/zk.xml/The_Library_Properties/org.zkoss.zul.theme.browserDefault
Should we consider to use the same property for this?